tests/cross/src/test/scala/tests/pc/CompletionScaladocSuite.scala

```diff
@@ -0,0 +1,194 @@
+package tests.pc
+
+import tests.BaseCompletionSuite
+
+object CompletionScaladocSuite extends BaseCompletionSuite {
+  check(
+    "scaladoc-methoddef",
+    """
+      |object A {
+      |  /**@@
+      |  def test(x: Int, y: Int): Int = ???
+      |}""".stripMargin,
+    """|/** */Scaladoc Comment
+       |""".stripMargin
+  )
+
+  check(
+    "scaladoc-classdef",
+    """
+      |object A {
+      |  /**@@
+      |  case class(x: Int) {}
+      |}""".stripMargin,
+    """|/** */Scaladoc Comment
+       |""".stripMargin
+  )
+
+  check(
+    "scaladoc-no-completion",
+    """
+      |object A {
+      |  /**@@
+      |}""".stripMargin,
+    ""
+  )
+
+  checkEdit(
+    "scaladoc-methoddef-edit",
+    """|object A {
+       |  /**@@
+       |  def test1(param1: Int, param2: Int): Int = ???
+       |  def test2(param1: Int, param2: Int, param3: Int): Int = ???
+       |}
+       |""".stripMargin,
+    """|object A {
+       |  /**
+       |  *
+       |  * @param param1 $0
+       |  * @param param2
+       |  * @return
+       |  */
+       |  def test1(param1: Int, param2: Int): Int = ???
+       |  def test2(param1: Int, param2: Int, param3: Int): Int = ???
+       |}
+       |""".stripMargin,
+    filter = _.contains("/** */")
+  )
+
+  checkEdit(
+    "scaladoc-classdef-edit",
+    """|object A {
+       |  /**@@
+       |  class Test1(param1: Int, param2: Int) {}
+       |  class Test2(param1: Int, param2: Int, param3: Int) {}
+       |}
+       |""".stripMargin,
+    """|object A {
+       |  /**
+       |  *
```

Comment:
The [official docs](https://docs.scala-lang.org/style/scaladoc.html#classes) also suggest the `@constructor` tag for classes. What do you think?

Reply:
I think that's a good idea:+1: I'm going to implement it later!

Reply:
done in https://github.com/scalameta/metals/pull/1250/commits/f4e93f44cc33c1fb3ff0a670e17575be5ca9d9f6